modules/oxide/src/less/theme/components/tag/tag.less

```
@@ -1,18 +1,53 @@
+@tag-border-width: 1px;
```

Stylelint (17.2.0)

[error] Unexpected unknown at-rule "@tag-border-width:" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)